.github/workflows/test-models.yml

```diff
@@ -126,12 +126,12 @@ jobs:
         export DYLD_LIBRARY_PATH=/usr/local/lib/
         echo $PATH
         pip list | grep -E "torch|iree"
-        pytest -s --ci --ci_sha=${SHORT_SHA} --local_tank_cache="/Volumes/builder/anush/shark_cache" tank/test_models.py -k vulkan --update_tank
+        pytest --ci --ci_sha=${SHORT_SHA} --local_tank_cache="/Volumes/builder/anush/shark_cache" -k vulkan --update_tank
```

Comment:
why does this work without a target for pytest?

Reply:
tank/test_models.py is the only valid target from the base SHARK directory -- the other ones are ignored via SHARK/pytest.ini